### PR TITLE
DCS-854 Fix security audit

### PR DESCRIPTION
**Pull request by andrewrlee**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -1,22 +1,20 @@
...
 
 orbs:
-  hmpps: ministryofjustice/hmpps@2.1.0
+  hmpps: ministryofjustice/hmpps@2.3.0
 
 executors:
   integration-tests:
     docker:
-      - image: circleci/node:14-buster-browsers
+      - image: circleci/node:14.15.3-buster-browsers
```
Comment (timharrison-moj) [inline .circleci/config.yml:9]: Might be preferable to use the `cimg/node: 14.15.3-browsers`  image here? The orb from v2.3 uses cimg/<<tag>> images here.

Comment (andrewrlee) [reply .circleci/config.yml:9]: ah, yes. thanks

Comment (timharrison-moj) [reply .circleci/config.yml:9]: Additional browser tools can be installed using the circle browser-tools orb if needed - though they have not been required in the other services I've done so far. If needed - add the orb, install the tools..
e.g.    
orbs:
  browser-tools: circleci/browser-tools@1.1.0

And 
  steps:
      - browser-tools/install-browser-tools
      - checkout
      - other tasks...

File: .circleci/config.yml [hunk 2]
```diff
@@ -1,22 +1,22 @@
...
     working_directory: ~/app
 
 jobs:
   build:
-    executor: builder
+    executor:
+      name: hmpps/node
+      tag: 14.15.3-browsers
     steps:
       - checkout
```
Comment (timharrison-moj) [inline .circleci/config.yml:21]: In this section there is still an update of npm, which can come out entirely - its pre-installed with the image at a compatible version. Its this section:
` - run:
        name: Update npm
        command: sudo npm install -g npm@latest
`

Comment (andrewrlee) [reply .circleci/config.yml:21]: Thanks - removed this too

**Review by timharrison-moj — APPROVED** [review]
LGTM

**Pull request merged**